clis/tcli/src/cli.rs

```diff
@@ -665,29 +689,25 @@ impl Cli {
                         Ok(udp_socket) => {
                             let udp_socket = Arc::new(udp_socket);
 
-                    // Timeout for stunner
-                    let sleep = sleep(Duration::from_secs(20));
-                    tokio::pin!(sleep);
-                    // let socket = udp_socket.clone();
-                    const MAX_PACKET: usize = 65536;
-                    let mut rx_buff = [0u8; MAX_PACKET];
-                    loop {
-                        tokio::select! {
-                            Ok((_len, _src_addr)) = udp_socket.recv_from(&mut rx_buff) => {
-                                match rx_buff.first().map(|b| PacketTypeRelayed::from(*b)) {
-                                    Some(PacketTypeRelayed::Pinger) => cli_res!(res; (i "Pinger message received")),
-                                    Some(PacketTypeRelayed::Ponger) => cli_res!(res; (i "Ponger message received")),
-                                    other => cli_res!(res; (i "Unexpected packet: {:?}", other)),
-                                }
-                                break;
-                            },
-                            () = &mut sleep => {
-                                cli_res!(res; (i "timeout"));
-                                break;
-                            },
-
-                        }
-                    }
+                            // Timeout for stunner
```

Comment:
Just to double-check, this is just reformatting, as previously the whole code block was off-by-one indented?

Reply:
:+1: